solutions/capacity-scenario/readme.md

```diff
@@ -0,0 +1,217 @@
+
+The link above sets up the pre-requisites to using the information oversharing template, which are:
+
+- Create a Synapse Workspace
+- Create a Spark Pool for the Synapse workspace
+- Create a storage account for the extracted data
+- Grant permission to the Synapse workspace & the MGDC Service Principal to the storage account as Blob Data Contributor
+- Create an Azure SQL Server
+- Create a sample database within the Azure SQL Server.
+
+By clicking on the above button (or navigating to the linked URL), users will be brought to the Azure portal on the Custom deployment page.
+
+On that screen, on top of providing information about the resource group and region to deploy the components into, they will need to provide the following information:
+
+- Application Id to be used by MGDC (from step #3, ending in 9826)
+- Application secret for that app
+- A new password for the Azure SQL Server
+
+Once all required information has been provided, click on the Review + create button at the bottom of the page:
+
+
+![](Images/11.png)
+
+This will validate that the information provided to the template is correct. Once the information has been validated, click on the Create button at the bottom of the page.
+
+![](Images/12.png)
+
+This will initiate the deployment. It should normally take about 5 minutes for the whole deployment to 
+complete.
+
+## Synapse pipeline template
+
+1.  After the pre-reqs are complete, navigate to the Synapse workspace just created
+
+![](Images/21.png)
+
+2.  Open the Synapse Studio
+
+![](Images/22.png)
+
+1.  Navigate to "Integrate -> Add new resource -> **Browse gallery**"
+
+![](Images/23.png)
+
+4.  Search for "SharePoint" and select the "Unlock Capacity Analytics and Insights for SharePoint and OneDrive using Microsoft 365 Datasets" template and Continue
+
+![](Images/24.png)
+
+5.  Create the new **Target** Linked services required by this pipeline
+
+![](Images/25.png)
+
+6.  Provide the parameters of the Linked Service
+    - Select Authentication Type = Service Principal 
+    - Use the storage account name, SPN id and secret (SPN key) from the pre-req steps above
+    - Test Connection and then click on Create
+
+![](Images/25a.png)
+
+
+7.  Repeat the Linked Service creation steps for the **source** Linked Service
+
+![](Images/25b.png)
+
+8.  Select "Open Pipeline"
+
+![](Images/26.png)
+
+
+9. Click on "Publish All" to validate and publish the pipeline
+
+![](Images/27.png)
+
+
+10. Review the changes and click Publish
+
+![](Images/27a.PNG)
+
+11. Verify that the pipeline has been successfully published
+
+![](Images/27b.PNG)
+
+12. Trigger the pipeline
```

Comment:
I assume we could add a note here: "Note: You should not run the pipeline until the Files dataset becomes available."